### PR TITLE
Basic API Information

### PR DESCRIPTION
**Pull request by Tantomile**
Here is the basic info for the API. Please review and let me know :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# API
+# TheCookieMC API
+Here you can find the documentation for TheCookieMC's API! All requests are sent through `api.thecookiemc.net`. The system currently only works with player UUIDs, not usernames. As the server is under active development, some API requests may be removed or changed without prior notice.
```
Comment (BillyDotWS) [inline README.md]: The welcome shouldn't be in the same place as api information.

Comment (BillyDotWS) [reply README.md]: The "without piror notice" isn't accurate - notice will be given in the discussions tab of github.

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+Here you can find the documentation for TheCookieMC's API! All requests are sent through `api.thecookiemc.net`. The system currently only works with player UUIDs, not usernames. As the server is under active development, some API requests may be removed or changed without prior notice. 
+### Available Values: 
+**{uuid}**: <br>
+A Player's 36 Character "Unique User ID". This can be found by searching a player on sites like NameMC. This does not reset after username changes. 
+<br>
+<br>**{game}**: <br>
+The Internal IDs of games on TheCookieMC.
+<br> Beacon Battle: `beacon_battle`
+<br> Trapped: `trapped`
+<br> Nether Raid: `nether_raid`
```
Comment (BillyDotWS) [inline README.md]: This should be in a table.

**Review by BillyDotWS — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Tantomile) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (Tantomile) [reply README.md]: Updated. Not sure what you mean by "The welcome shouldn't be in the same place as api information."

**Comment by Tantomile** [comment]
- Added configuration for documentation in README.md to be viewed via web address (Currently available at https://tantomile.github.io/CookieMCAPI/, after merge should be visible at thecookiemc.github.io/api). Minor formatting issues on webpage will be fixed later this week

**Comment by Tantomile** [comment]
Since none of these games are going to be used in the way planned at the creation of this PR, i'm going to go ahead and close this request.

**Pull request closed without merging**